kustomize/base/quay.deployment.yaml

```diff
@@ -7,7 +7,6 @@ metadata:
   annotations:
     quay-component: base
 spec:
-  replicas: 2
```

Comment:
Is there a specific reason for lowering the amount of replicas?

Reply:
If not removed then we will have a concurrency problem with the HPA as the we can't assume the number of replicas set by the HPA (assume for instance that the HPA set this field to `3`, we can't reset it back to `2`). i.e. if we don't set it keeps the value set by the HPA.

Reply:
After discussing this with @ibazulic I have restored the old behavior with a small difference: only set the number of replicas to 2 if HPA is not managed. Can you resolve this if you are OK with it?